src/library_pthread.js

```diff
@@ -939,6 +939,9 @@ var LibraryPThread = {
 #if PTHREADS_DEBUG
     dbg('exitOnMainThread');
 #endif
+#if PROXY_TO_PTHREAD
+    {{{ runtimeKeepalivePop() }}};
+#endif
```

Comment:
I know I have another PR out that completely removes this function..  too many conflicting changes right now.  I think this one can land first.